clusterctl/testdata/delete-cluster-no-args-invalid-flag.golden

```diff
@@ -3,8 +3,12 @@ Usage:
   clusterctl delete cluster [flags]
 
 Flags:
+      --cluster string               The name of the kubeconfig cluster to use
   -h, --help                         help for cluster
+      --kubeconfig string            Path to the kubeconfig file to use for connecting to the cluster to be deleted, if empty, the default KUBECONFIG load path is used.
+  -n, --namespace string             If present, the namespace scope for this CLI request
```

Comment:
This is namespace for the kubectl apply commands, which isn't super clear from the help text.